docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -63,11 +63,21 @@ This will be returned [via a webook](/medical-api/more-info/webhooks#medical-rec
 
 ## Request Body
 
-<BodyField name="meta" type="object" optional>
+<BodyField name="metadata" type="object" optional>
```

Comment:
Could move this to a snipped so we have it written only once - any maintenance is less likely to introduce errors.